include/Configuration.hpp

```diff
@@ -101,12 +101,17 @@ class Configuration {
     std::size_t nAdditionalInstantiations = 0;
   };
 
+  struct ZX {
+    bool setAllAncillasGarbage = false;
+  };
```

Comment:
I believe this cannot be a ZX-only option as this would create an unsound overall equivalence checker.
This is nicely demonstrated by a test to be added. Using the first example with a single ancillary qubit, that is not garbage, and a single X gate in one of the circuits, the ZX checker with the new option would report equivalent, while the DD checkers, which can handle the non-garbage output qubit, would report non-equivalent.
When running the checks in parallel (the default), the order in which they finish would determine the result.
This is clearly undesirable. Thus, the option needs to be more general and apply to all checkers. As such, it probably fits best in the "execution" category of the configuration options.

Reply:
That is something I had not considered, but you are obviously right. Having it be a global setting also simplifies the logic quite a bit since nothing needs to be done in addition to setting the ancillaries to garbage before starting any checker.